googleanalytics/userReport.py

```diff
@@ -0,0 +1,116 @@
+import os
+from superagi.tools.base_tool import BaseTool
+from pydantic import BaseModel, Field
+import json
+from typing import Type, Optional
+from superagi.resource_manager.file_manager import FileManager
+import yaml
+from google.analytics.data_v1beta import BetaAnalyticsDataClient
+from google.analytics.data_v1beta.types import (
+    DateRange,
+    Dimension,
+    Metric,
+    RunReportRequest,
+)
+
+class UserReportInput(BaseModel):
+    start: str = Field(..., description="The starting date of the query, in YYYY-MM-DD format")
+    end: str = Field(..., description="The last date of the query, in YYYY-MM-DD format, if today, return today's date")
+    issinglefile: bool= Field(..., description="True if it is specified that the report should be stored in a single file. False otherwise.")
+
+
+class reportTool(BaseTool):
```

Comment:
rename the class to to GoogleAnalyticsReportTool and
GoogleAnalyticsReportToolInput